packages/admin/cms-admin/src/generator/future/generateForm.ts

```diff
@@ -194,6 +194,12 @@ export function generateForm(
         });
     }
 
+    const finalFormSubscription = [...(generatedFields.finalFormConfig?.subscription?.values ? ["values"] : [])];
+    const finalFormRenderProps = [
+        ...(generatedFields.finalFormConfig?.renderProps?.values ? ["values"] : []),
+        ...(generatedFields.finalFormConfig?.renderProps?.form ? ["form"] : []),
+    ];
+
```

Comment:
Maybe make subscription and renderProps string arrays instead and pass them here?

Reply:
switched it to a more generic forwarding of the object and not to array because with array I would have to make sure values are unique.